NEWS.md

```diff
@@ -237,6 +237,36 @@ Outstanding ones only.
         p C.ancestors #=> [C, M1, M2, Object, Kernel, BasicObject]
         ```
 
+    * Module#public, Module#protected and Module#private methods now accept single
+      array (or array like) argument with a list of method names.
+      [[Feature #17314]]
+
+        ```ruby
+        class Foo
+          def foo; end
+          def bar; end
+
+          private [:foo, :bar]
+        end
+        ```
+
+    * Module#attr_accessor, Module#attr_reader, Module#attr_writer and Module#attr
+      methods now return array of defined methods names as symbols (previously
+      they ).
+      You can mix this with `public/protected/private` methods accepting array.
+      [[Feature #17314]]
+
+        ```ruby
+        class Foo
+          attr_accessor :foo, :bar #=> [:foo, :foo=, :bar, :bar=]
+          private attr_accessor :baz
+        end
+        ```
+
+    * Module#alias_method now returns defined alias as symbol (previously it
```

Comment:
"now returns *the* defined alias as symbol", remove "(previously..."

Reply:
Does it apply to all places where I used expression "of defined methods"?

Reply:
No, others are fine (plural), thanks